src/session.js

```diff
@@ -202,16 +202,17 @@ ghostdriver.Session = function(desiredCapabilities) {
         if (k.indexOf(_capsPageWhitelistPref) === 0) {
             const pageWhitelist = [];
             const len2 = desiredCapabilities[k].length;
-            for(var i = 0; i < len; i++) {
+            for(var i = 0; i < len2; i++) {
```

Comment:
I think we need to rename those `len` and `len2` to indicate precisely what are they length of.
This would have probably avoided the issue in the first place.